webapp/channels/src/components/new_channel_modal/new_channel_modal.tsx

```diff
@@ -217,48 +197,14 @@ const NewChannelModal = () => {
         }
     };
 
-    const handleOnDisplayNameChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        e.preventDefault();
-        const {target: {value: displayName}} = e;
+    const handleOnURLChange = useCallback((updatedURL: string) => {
```

Comment:
0/5, but perhaps we could just pass `setURL` into `ChannelNameFormField` directly instead of making a separate callback for this? I've noticed a few places in this PR where we might be able to get rid of the extra callback, but I could also understand leaving it as-is so that it matches other code where we do more in these callbacks